server/db/driver/pg/orders.go

```diff
@@ -0,0 +1,584 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package pg
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/decred/dcrdex/server/account"
+	"github.com/decred/dcrdex/server/db"
+	"github.com/decred/dcrdex/server/db/driver/pg/internal"
+	"github.com/decred/dcrdex/server/market/types"
+	"github.com/decred/dcrdex/server/order"
+	"github.com/lib/pq"
+)
+
+type Error string
+
+func (err Error) Error() string { return string(err) }
+
+const (
+	ErrUnknownOrder = Error("unknown order")
+)
+
+// utxo implements order.UTXO
+type utxo struct {
+	txHash []byte
+	vout   uint32
+}
+
+func (u *utxo) TxHash() []byte { return u.txHash }
+func (u *utxo) Vout() uint32   { return u.vout }
```

Comment:
We keep redefining simple `utxo` types, so we may wish to put this in `server/market/types` or `server/dex/types`.